content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment. There are four levels of threading supported by
+the \openshmem{} implementation.
+ 
+\begin{itemize}
+\item {\bf \CONST{SHMEM\_THREAD\_SINGLE}} The \openshmem{} program may not be multithreaded, 
+and all \openshmem{} interfaces are invoked by a single thread. 
+
+\item {\bf \CONST{SHMEM\_THREAD\_FUNNELED}}
+The \openshmem{} program may be multithreaded, however, the 
+program must ensure that only the main thread invokes the \openshmem{}
```

Comment:
Main thread is not defined, propose adding "(i.e. the thread that called shmem_init_thread)".